src/expr.rs

```diff
@@ -53,11 +55,37 @@ impl Rewrite for ast::Expr {
             ast::Expr_::ExprTup(ref items) => {
                 rewrite_tuple_lit(context, items, self.span, width, offset)
             }
+            ast::Expr_::ExprLoop(ref block, _) => {
+                // FIXME: this drops any comment between "loop" and the block.
+                block.rewrite(context, width, offset).map(|result| {
```

Comment:
I don't think so. Blocks go to the new line immediately after the opening brace.. That's why `width` and `offset` are disregarded in the implementation of `Rewrite` for `ast::Block`. It only needs the current block indentation.